.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Summary
+
+<!-- Please reference the issue this PR addresses. -->
+Addresses issue #
+
+## Checklist
+
+- [ ] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
+- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
+- [ ] My code follows the [coding standards](https://github.com/ampproject/amp-wp/contributing/engineering.md#coding-standards).
+- [ ] My code has proper inline documentation.
```

Comment:
Linting should catch this. If you omit PhpDoc for a function, then PHPCS gives a `Squiz.Commenting.FunctionComment.Missing` error. If you do not provide a `@param` tag, that's a `Squiz.Commenting.FunctionComment.MissingParamTag` error. And so on. So perhaps this is also redundant?

Reply:
The idea here is beyond PHP doc, commenting code explaining why something is done in a certain way, or if something fixes a bug etc. within the code is really useful for other devs. Perhaps we should have a section for that in the Engineering doc.

In fact, I am modifying the checkboxes list to just have "My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/contributing/engineering.md)" which encompass everything.